apps/CatalystApp/CatalystApp/AVPlayerViewSwiftUi.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AVPlayerViewSwiftUi.swift
+//  CatalystApp
+//
+//  Created by Emily Dixon on 10/28/22.
+//
+import Foundation
+import SwiftUI
+import AVKit
+
+/**
+ * Wraps an `AVPlayerViewController` with SwiftUI, providing accwss to its `AVPlayer`
+ */
+struct AVPlayerViewSwiftUi : UIViewControllerRepresentable {
```

Comment:
This is the tiniest nit 😅 But since this is also an example app, I think `PlayerView` might be clearer? It's clearly using SwiftUI & AVKit so feel like this doesn't need to be in the name, plus I thought SwiftUI components tend to be named more generally. I also have no production experience with SwiftUI so please take this with a grain of salt

Reply:
I do like that way better!